extension/src/extension.ts

```diff
@@ -168,9 +173,12 @@ export class Extension extends Disposable implements IExtension {
       new Setup(
```

Comment:
I am going to move a big chunk of the logic from the extension into `Setup` in a follow-up.

Reply:
I will increase test coverage when I make that change.